### PR TITLE
Disallow filenames beginning with "../../../"

### PR DESCRIPTION
**Pull request by CodeMicah**
Added a check when a file is created using the "New File" button for paths beginning in "../../../" to help prevent escaping the MC workspace.  

## Description
Simply check if the file name begins in "../../../" and don't create the file if it does.  

## Motivation and Context

Addresses issue #92

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My changes require updating to the plugin documentation.

### CODE REVIEW

**Comment by solvedDev** [comment]
Thanks a lot for the contribution!
However, the proposed implementation doesn't actually fix the problem of escaping the MC workspace.
I would propose using a RegExp like `/\.\.(\\|\/)/g` to detect the "../" pattern and then prevent the creation of a file

**Comment by CodeMicah** [comment]
So changing this to be if there are any ".."s instead of more than two?

**Comment by CodeMicah** [comment]
Sorry about the indentation.  I'm trying to fix it, vscode is being naughty.  
Any advice?

**Comment by solvedDev** [comment]
I‘d recommend installing the "Prettier" extension. We have the repo setup so that the code is formatted correctly automatically then :)

File: app/renderer/windows/CreateFile.js [hunk 1]
```diff
@@ -45,23 +45,24 @@ class FileContent {
...
 
-							if (
-								val === '' &&
-								!this.parent.actions[1].is_disabled
-							) {
+						if (
+							val === '' ||
+							/\.\.(\\|\/)/g.test(val)
+						) {
+							if (!this.parent.actions[1].is_disabled) {
```
Comment (solvedDev) [inline app/renderer/windows/CreateFile.js:65]:
```suggestion
						if (
							(val === '' ||
							/\.\.(\\|\/)/g.test(val)) &&
							!this.parent.actions[1].is_disabled
						) {
```

**Comment by CodeMicah** [comment]
That's great, thanks! I didn't have time this weekend to do it.

**Comment by CodeMicah** [comment]
Correct me if I'm wrong, but I think that if you put the `!this.parent.actions[1].is_disabled` in the if statement, then if the field is already disabled then it will enable it, causing the field to alternate enabled and disabled each time a change is made.  That's why I put a new statement in.

**Comment by CodeMicah** [comment]
Prettier is minimizing the imports at the beginning of the file. Do I need to change a setting?

**Comment by solvedDev** [comment]
The enabling/disabling scenario you are describing is prevented by the fact that it‘s an `if... else if...` statement and not an `if... else...` statement. Meaning that the button state only alternates if there is a change to be made

**Pull request merged**